packages/side-nav/src/constants.js

```diff
@@ -0,0 +1,68 @@
+const constants = {
+  sideNavMaxWidth: "280px"
+};
+
+// from packages/styles/tokens/breakpoints.scss
+const _breakpoints = {
+  phone: 0,
+  tablet: 769,
+  laptop: 1281,
+  desktop: 1681,
+  large: 1921
+};
+
+// from packages/styles/mixins/breakpoints.scss
+const mq = Object.keys(_breakpoints).reduce(
+  (obj, key) =>
+    Object.assign(obj, { [key]: `@media (min-width: ${_breakpoints[key]}px)` }),
+  {}
+);
+
+function reset(styles) {
```

Comment:
This and the `typographyBase` are the equivalent of the css mixins. I'd love to know if anyone has found better ways of implementing these versus just copying them.